libV2/server/middleware.js

```diff
@@ -39,9 +40,18 @@ function loggerMiddleware(req, res, next) {
 // next is purposely not called as all error responses are handled here
 // eslint-disable-next-line no-unused-vars
 function errorMiddleware(err, req, res, next) {
-    const code = err.code || 500;
+    let code = err.code || 500;
     let message = err.message || 'Internal Error';
 
+    // failed request validation by oas-tools
+    if (err.failedValidation) {
+        // You can't actually use destructing here
```

Comment:
why not?

Reply:
```
Tmac@Kennuc: /tmp cat test.js
let x;
let y;

{ x, y } = { x:1, y:2}
Tmac@Kennuc: /tmp node test.js
/tmp/test.js:4
{ x, y } = { x:1, y:2}
         ^

SyntaxError: Unexpected token '='
    at wrapSafe (internal/modules/cjs/loader.js:1054:16)
    at Module._compile (internal/modules/cjs/loader.js:1102:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
```

Reply:
hmm seems like it doesn't like destructuring onto existing objects... good to know. Not sure if the comment is required but yea.

Reply:
Note that from the node REPL it works just fine which is confusing...
```shell
Welcome to Node.js v14.1.0.
Type ".help" for more information.
> let x, y;
undefined
> { x, y } = { x:1, y:2 }
{ x: 1, y: 2 }
> console.log(x)
1
undefined
> console.log(y)
2
undefined
```